cypress/e2e/Examples/Demo.cy.js

```diff
@@ -16,10 +20,10 @@ describe('AsertionsSuite', () => {
 
     // //   expect(pageTitle).to.be.equal('Kitchen Sink')
     // assert.equal(pageTitle,'Kitchen Sink',"Page Title not matched." )
+    cy.visit('https://demoqa.com/elements')
 
```

Comment:
use the visit command above the 6th line, then only application will be launch.

Reply:
Changes done.